ios/Capacitor/Capacitor/CapacitorBridge.swift

```diff
@@ -207,6 +213,7 @@ internal class CapacitorBridge: NSObject, CAPBridgeProtocol {
         do {
             try JSExport.exportCapacitorGlobalJS(userContentController: webViewDelegationHandler.contentController,
                                                  isDebug: isDevMode(),
```

Comment:
Unrelated to the change, but since `isDevMode` is deprecated, we should use `isDevEnvironment`

Reply:
Oh, good catch.